src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -21,13 +21,19 @@
 
 class ActivityTensorFlowModel(TensorFlowModel):
 
-    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len):
+    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len, config):
 
+        _utils.suppress_tensorflow_warnings()
         self.gpu_policy = _utils.TensorFlowGPUPolicy()
         self.gpu_policy.start()
 
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
+        else:
+            seed=None
```

Comment:
Let's follow PEP8 here and surround the assignment with spaces. https://www.python.org/dev/peps/pep-0008/